src/AccessibilityInsights.SharedUx/ViewModels/ApplicationSettingsViewModel.cs

```diff
@@ -119,17 +119,17 @@ public int HighlighterMode
             }
         }
 
-        bool _playingScanningSound;
-        public bool PlayScanningSound
+        int _soundMode;
```

Comment:
naming nit: I'd suggest `_soundFeedbackMode` to emphasize that this is a backing field